File: MaratoneiA1/app/src/main/java/com/example/manoel/maratoneia1/ResultsMovie/DataMovie.java

package com.example.manoel.maratoneia1.ResultsMovie;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class DataMovie {

    @SerializedName("page")
    @Expose
    private Integer page;
    @SerializedName("results")
    @Expose
    private List<ResultMovie> results = null;
    @SerializedName("date")
    @Expose
    private Date date;
    @SerializedName("total_pages")
    @Expose
    private Integer totalPages;
    @SerializedName("total_results")
    @Expose
    private Integer totalResults;

    public Integer getPage() {
        return page;
    }

    public void setPage(Integer page) {
        this.page = page;
    }

    public List<ResultMovie> getResults() {
        return results;
    }

    public void setResults(List<ResultMovie> results) {
        this.results = results;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public Integer getTotalPages() {
        return totalPages;
    }

    public void setTotalPages(Integer totalPages) {
        this.totalPages = totalPages;
    }

    public Integer getTotalResults() {
        return totalResults;
    }

    public void setTotalResults(Integer totalResults) {
        this.totalResults = totalResults;
    }

}

File: MaratoneiKotlin/app/src/main/java/com/example/manoeltoj/maratoneikotlin/Movie/dataMovie/DataMovie.kt
package com.example.manoeltoj.maratoneikotlin.Movie.dataMovie

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class DataMovie{
    @SerializedName("page")
    @Expose
    var page:Integer? = null
        get() = field
        set(value){
            field = value
        }
    @SerializedName("results")
    @Expose
    var results:List<ResultMovie>? = null
        get() = field
        set(value) {
            field = value
        }
    @SerializedName("date")
    @Expose
    var date:Date? = null
        get() = field
        set(value) {field = value}
    @SerializedName("total_pages")
    @Expose
    var totalPages:Integer? = null
        get() = field
        set(value) {
            field = value
        }
    @SerializedName("total_results")
    @Expose
    var totalResults:Integer? =null
        get() = field
        set(value) {
            field = value
        }
}